plugins/modem-manager/fu-mm-utils.c

```diff
@@ -46,6 +46,23 @@ fu_mm_utils_get_udev_port_info (GUdevDevice	 *device,
 		g_set_object (&parent, next);
 	}
 
+	if (!device_sysfs_path && g_strcmp0 (g_udev_device_get_subsystem (device), "wwan") == 0) {
+		parent = g_udev_device_get_parent (device);
+		while (parent != NULL) {
```

Comment:
does `g_udev_device_get_parent_with_subsystem()` work better here?

Reply:
Oh, I don't know, can you give me some suggestions?